docs/create-packages/creating-a-package-dotnet-cli.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Create a NuGet package using the dotnet CLI
+description: A detailed guide to the process of designing and creating a NuGet package, including key decision points like files and versioning.
+author: karann-msft
+ms.author: karann
+ms.date: 07/09/2019
+ms.topic: conceptual
+---
+
+# Create a NuGet package using the dotnet CLI
+
+No matter what your package does or what code it contains, you use one of the CLI tools, either `nuget.exe` or `dotnet.exe`, to package that functionality into a component that can be shared with and used by any number of other developers. This article describes how to create a package using the dotnet CLI. To install the `dotnet` CLI, see [Install NuGet client tools](../install-nuget-client-tools.md). Starting in Visual Studio 2017, the dotnet CLI is included with .NET Core workloads.
+
+For .NET Core and .NET Standard projects that use the [SDK-style format](../resources/check-project-format.md), and any other SDK-style projects, NuGet uses information in the project file directly to create a package. For detailed steps, see [Create .NET Standard Packages with dotnet CLI](../quickstart/create-and-publish-a-package-using-the-dotnet-cli.md), [Create .NET Standard Packages with Visual Studio](../quickstart/create-and-publish-a-package-using-visual-studio.md) or [NuGet pack and restore as MSBuild targets](../reference/msbuild-targets.md).
+
+`msbuild -t:pack` is functionality equivalent to `dotnet pack`. To build with MSBuild, the concepts are the same as described in this article, but the command line commands are slightly different.
+
+> [!IMPORTANT]
+> This topic applies to [SDK-style](../resources/check-project-format.md) projects, typically .NET Core and .NET Standard projects.
+
+## Set properties
+
+The following properties are required to create a package.
+
+- `PackageId`, the package identifier, which must be unique across the gallery that hosts the package.
+- `Version`, a specific version number in the form *Major.Minor.Patch[-Suffix]* where *-Suffix* identifies [pre-release versions](prerelease-packages.md)
+- The package title as it should appear on the host (like nuget.org)
+- `Authors`, author and owner information.
```

Comment:
This is technically correct, but all of these properties have defaults :) 

So even a project as simple as: 

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>netstandard2.0</TargetFramework>
  </PropertyGroup>

</Project>
```
will pack since: 

PackageId defaults to AssemblyName. 
Version defaults to 1.0.0
Authors and Company default to AssemblyName. 

Not sure if/how we want to include that information.

Reply:
Thx! I'll mention the defaults.